modules/n1ql/pages/n1ql-language-reference/jsonfun.adoc

```diff
@@ -1,26 +1,34 @@
 = JSON Functions
-:description: DECODE_JSON(expression)
+
+:description: Functions for manipulating data in JSON.
+
 :page-topic-type: reference
 
 {description}
 
+== DECODE_JSON(expression)
+
+=== Description
 Unmarshals the JSON-encoded string into a {sqlpp} value.
 The empty string is MISSING.
 
-ENCODE_JSON(expression)
+== ENCODE_JSON(expression)
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function